app/controllers/conference_registrations_controller.rb

```diff
@@ -1,10 +1,14 @@
 class ConferenceRegistrationsController < ApplicationController
   before_filter :authenticate_user!, except: [:new, :create]
   load_resource :conference, find_by: :short_title
-  authorize_resource :conference_registrations, class: Registration
+  authorize_resource :conference_registrations, class: Registration, except: [:new, :create]
   before_action :set_registration, only: [:edit, :update, :destroy, :show]
 
   def new
+    @registration = Registration.new(conference_id: @conference.id)
+    # make sure that conference is open for registration
```

Comment:
That is not exactly what the following line does. The following does checkes the authorization of @registration, it checks if there is proper access to the action 'new' for @registration. Checking if the registration of the conference is open, is one of the defined abilities but there are / there can be more. So I think the comment is a bit misleading.